### PR TITLE
Update Colour Scheme

### PR DESCRIPTION
**Pull request by rishiosaur**
This updates the body font & the colors to the spec that was discussed on the Discord.

### CODE REVIEW

**Comment by rishiosaur** [comment]
image

^ What passerine.io looks like now! @slightknack you'll need to update the logo as well

**Comment by slightknack** [comment]
I'll update the logo as soon as possible :)

File: index.html [hunk 1]
```diff
@@ -12,11 +12,11 @@
   <div class="content">
     <div class="reasons">
       <div class="section">
-        <h1 style="font-weight: normal;">Passerine <a href="https://github.com/vrtbl/passerine"><img src="https://raw.githubusercontent.com/vrtbl/passerine/master/Logo.png" class="inline"></img></a></h1>
-        <h3>A small extensible programming language designed for concise expression with little code.</h3>
+        <h1 style="font-weight: bold;">Passerine <a href="https://github.com/vrtbl/passerine"><img src="https://raw.githubusercontent.com/vrtbl/passerine/master/Logo.png" class="inline"></img></a></h1>
```
Comment (slightknack) [inline index.html:15]: A bit on the fence about this 🤔 (changing font weight to bold) - but that's something for another time.

**Review by slightknack — APPROVED** [review]
Everything looks really good, nice job!

**Pull request merged**